ch32v003fun/ch32v003fun.h

```diff
@@ -1222,6 +1222,13 @@ typedef struct
     __IO uint32_t EXTEN_CTR;
 } EXTEN_TypeDef;
 
+/* The reference manual for the 32v2xx/v3xx refernce this as "CTR" on "EXTEND" so adding an alias here. for any processor. */
```

Comment:
Typos:
- "refernce" should be ”reference"
- "on" should be "or"?
- "32v2xx" should be "ch32v2xx"?
- ". for any processor" doesn't seem to read properly?

Reply:
Fixed in new rev.  Thanks for the feedback.  I will wait for it to pass CI.